src/fusion.cpp

```diff
@@ -512,12 +548,24 @@ miopenStatus_t ConvForwardOpDescriptor::GetOutputDesc(TensorDescriptor& output_d
         [&]() { output_desc = base_desc.GetForwardOutputTensor(input_desc, filter_desc); });
 }
 
+/*
```

Comment:
dead code?

Reply:
@amberhassaan maybe kept for debugging purposes?

Reply:
yes, this should be removed. @iq136boy commented it out when I wasn't paying attention :). 
In reality, we need to deduce the pointer type from `ConvolutionDescriptor` and not assume `float` all the time, but for now CK seems to support `float` only for these scaling factors.